src/Board/index.jsx

```diff
@@ -0,0 +1,215 @@
+    checkIfWonHorizontally();
+    checkIfWonVertically();
+    checkIfWonDiagonally();
+    checkIfDraw();
+  };
+
+  const checkConditionsToWin = (array) => {
+    return allEqual(array) && array[0].value !== null;
+  };
+  const allEqual = (array) =>
+    array.every((object) => object.value === array[0].value);
+
+  const checkIfWonHorizontally = () => {
+    for (const row of Object.values(board)) {
+      if (checkConditionsToWin(row)) {
+        setGameStatus(GameStatus.gameOver);
+        setH1Title(H1Title.playerWon);
+        showWinner();
+      }
+    }
+  };
+
+  const checkIfWonVertically = () => {
+    for (let i = 0; i < Object.keys(board).length; i++) {
+      let horizontalResultArray = [];
+      for (const row of Object.values(board)) {
+        horizontalResultArray.push(row[i]);
+      }
+      if (checkConditionsToWin(horizontalResultArray)) {
+        setGameStatus(GameStatus.gameOver);
+        setH1Title(H1Title.playerWon);
+        showWinner();
+      }
+    }
+  };
+
+  const checkIfWonDiagonally = () => {
+    const rightDiagonalResultArray = [
+      Object.values(board)[0][0],
+      Object.values(board)[1][1],
+      Object.values(board)[2][2],
+    ];
+    if (checkConditionsToWin(rightDiagonalResultArray)) {
+      setGameStatus(GameStatus.gameOver);
+      setH1Title(H1Title.playerWon);
+      showWinner();
+    } else {
+      const leftDiagonalResultArray = [
+        Object.values(board)[0][2],
+        Object.values(board)[1][1],
+        Object.values(board)[2][0],
+      ];
+      if (checkConditionsToWin(leftDiagonalResultArray)) {
+        setGameStatus(GameStatus.gameOver);
+        setH1Title(H1Title.playerWon);
+        showWinner();
+      }
+    }
+  };
+
+  const checkIfDraw = () => {
+    let allResultsArray = [];
+    for (const row of Object.values(board)) {
+      allResultsArray.push(row);
+    }
+    allResultsArray = allResultsArray.flat();
+    allResultsArray = allResultsArray.filter((result) => result.value === null);
+    if (allResultsArray.length === 0) {
+      setGameStatus(GameStatus.draw);
+      setH1Title(H1Title.draw);
+      showDraw();
+    }
+  };
+
+  const getFieldById = (fieldIdToFind) => {
+    for (const rowKey in board) {
+      const rowArray = board[rowKey];
+      const foundField = rowArray.find((field) => field.id === fieldIdToFind);
+      if (foundField) {
+        return foundField;
+      }
+    }
+  };
+
+  const move = (fieldId) => {
+    const field = getFieldById(fieldId);
+    if (!field.disabled && turn === Turn.playerOne) {
+      field.value = turn;
+      setBoard({ ...board });
+      setTurn(Turn.playerTwo);
+      setArrowToSecondPlayer();
+      checkIfWonOrDrawAndSetGameStatus();
+    } else if (!field.disabled && turn === Turn.playerTwo) {
+      field.value = turn;
+      setBoard({ ...board });
+      setTurn(Turn.playerOne);
+      setArrowToFirstPlayer();
+      checkIfWonOrDrawAndSetGameStatus();
+    }
+  };
```

Comment:
First of all, this function should look like this:
```suggestion
  const move = (fieldId) => {
    const field = getFieldById(fieldId);
    field.value = turn;

    if (!field.disabled && turn === Turn.playerOne) {
      setTurn(Turn.playerTwo);
      setArrowToSecondPlayer();
    } else if (!field.disabled && turn === Turn.playerTwo) {
      setTurn(Turn.playerOne);
      setArrowToFirstPlayer();
    }
     setBoard({ ...board });
     checkIfWonOrDrawAndSetGameStatus();
  };
```
But…
Thanks to simplifying the state by removing `arrowFirstPlayerClass` and `arrowSecondPlayerClass`, this function will be simplified even more. It is up to you to finish this code simplification.

Reply:
done